src/frontend/apps/desk/src/app/__tests__/page.test.tsx

```diff
@@ -16,6 +16,8 @@ describe('Page', () => {
       useAuthStore.setState({ authenticated: true });
     });
 
-    expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent('Desk');
+    expect(screen.getByRole('heading', { level: 1 })).toHaveTextContent(
+      'Hello Desk !',
+    );
```

Comment:
some issue with your rebase?

Reply:
No, this part was testing a text from the Header component, but the Header moved to `InnerLayout`, so I needed to adjust my test.